internal/bminventory/inventory.go

```diff
@@ -2181,6 +2167,54 @@ func (b *bareMetalInventory) UploadHostLogs(ctx context.Context, params installe
 	return installer.NewUploadHostLogsNoContent()
 }
 
+func (b *bareMetalInventory) DownloadHostLogs(ctx context.Context, params installer.DownloadHostLogsParams) middleware.Responder {
```

Comment:
I don't mind if you do it in a separate JIRA, but we need to add support for downloading logs via presigned URL as well

Reply:
it will be another api? or same one?